app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,47 @@
+module DashboardHelper
+  def value_resources_total_chart projects, year
+    hash_month = ProjectsMonthSummaryAnalyzer.call(projects, year).result
+    # not use total in chart
+    hash_month.delete(:total)
+    values = {}
+    resources = {}
+    diffs = {}
+    hash_month.each do |key, value|
+      month_string = [key, year].join "-"
+      values[month_string] = value[:value]
+      resources[month_string] = value[:resource]
+      diffs[month_string] = value[:diff]
+    end
+    # chart array data
+    [{name: t("value_resources.index.value"), data: values},
+     {name: t("value_resources.index.resource"), data: resources},
+     {name: t("value_resources.index.diff"), data: diffs}]
+  end
+
+  def project_statistics_by_status projects
+    project_total = projects.count
+    Project.statuses.map do |key, value|
+      status_count = projects.where(status: key).count
+      ratio = status_count / project_total.to_f
+      percentage = (ratio *  Settings.number.value_100)
```

Comment:
```suggestion
      percentage = (ratio *  100)
```
tính % thôi thì chắc để luôn 100 cx được a nhể

Reply:
anh nghĩ là nên dùng setting thì cx oke mà, để chị xem thế nào

Reply:
oke để luôn 100 cũng dc vì con số này k thay đổi